### PR TITLE
Allow final version of React 15

### PR DESCRIPTION
**Pull request by martijnrusschen**
Note: I removed React 0.13 from the package.json since I needed to include the separate add-on for TestUtils. Not sure if there's a better solution.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,8 @@
     "webpack-dev-server": "^1.8.2"
   },
   "peerDependencies": {
-    "react": ">=0.13.2 || ^0.14.0-rc1 || ^15.0.0-rc"
+    "react": "^0.14.0 || ^15.0.0",
+    "react-addons-test-utils": "^0.14.0 || ^15.0.0"
```
Comment (rafeememon) [inline package.json]: This should be a `devDependency` instead

File: package.json [hunk 2]
```diff
@@ -49,7 +50,7 @@
     "webpack-dev-server": "^1.8.2"
   },
   "peerDependencies": {
-    "react": ">=0.13.2 || ^0.14.0-rc1 || ^15.0.0-rc"
```
Comment (gaearon) [inline package.json]: To be pedantic, 15 stable is *already* allowed. `^15.0.0-rc` includes any stable `15` version.

Comment (martijnrusschen) [inline package.json]: Good point. Up to you, I can abandon this or just push it through if we want to exclude RC versions.

**Comment by davidpfahler** [comment]
LGTM. @gaearon? I'm fine with this.

**Comment by gaearon** [comment]
Sure.

**Comment by davidpfahler** [comment]
Rebased to master.

**Pull request closed without merging**